stdlib/public/core/Arrays.swift.gyb

```diff
@@ -151,7 +151,7 @@ if True:
 /// Here's an implementation of those steps:
 ///
 ///     if let i = absences.index(where: { $0 > 0 }) {                      // 1
-///         let absencesAfterFirst = absences.suffix(from: i + 1)           // 2
+///         let absencesAfterFirst = absences[(i + 1)...]                   // 2
```

Comment:
None of those syntaxes work for me; the parens seem to be necessary. We _could_ add a `+` overload for a `CountablePartialRangeFrom<Bound>` and `Bound.Stride` okay I'll see myself out…